integrations/access/accessrequest/app.go

```diff
@@ -352,16 +352,23 @@ func (a *App) getMessageRecipients(ctx context.Context, req types.AccessRequest)
 		recipientSet.Add(common.Recipient{})
 		return recipientSet.ToSlice()
 	case types.PluginTypeOpsgenie:
-		if recipients, ok := req.GetSystemAnnotations()[types.TeleportNamespace+types.ReqAnnotationSchedulesLabel]; ok {
-			for _, recipient := range recipients {
-				rec, err := a.bot.FetchRecipient(ctx, recipient)
-				if err != nil {
-					log.Warning(err)
-				}
-				recipientSet.Add(*rec)
+		// For backwards compatibility we need to use the schedules label to grab recipients
+		// if the notify services isn't set.
+		recipients, ok := req.GetSystemAnnotations()[types.TeleportNamespace+types.ReqAnnotationNotifyServicesLabel]
+		if !ok {
+			recipients, ok = req.GetSystemAnnotations()[types.TeleportNamespace+types.ReqAnnotationSchedulesLabel]
+			if !ok {
+				return recipientSet.ToSlice()
+			}
+		}
+		for _, recipient := range recipients {
+			rec, err := a.bot.FetchRecipient(ctx, recipient)
+			if err != nil {
+				log.Warning(err)
 			}
-			return recipientSet.ToSlice()
+			recipientSet.Add(*rec)
```

Comment:
If there was an error, doesn't that mean that `rec` is likely nil? (In which case this code would panic..)